.NET Guide - ReSharper Essentials (#153)
JetBrains/jetbrains_guide: sites/dotnet-guide/contents/tutorials/resharper-essentials/refactoring/index.md

---
type: TutorialStep
date: 2021-03-04
title: Refactoring Code with ReSharper
technologies: [.net, csharp]
topics: [refactoring]
products: [rider]
author: rachelappel
subtitle: How to safely refactor code, without breaking the build? Let's have a look.
thumbnail: ./thumbnail.png
longVideo: 
    poster: ./poster.png
    url: https://TODO
---

### Refactoring

> Refactoring is the process of restructuring code to improve it. The improvements may be making the code more readable, formatting the code, or organizing it better.

Reviewer: ```suggestion
Refactoring is the process of restructuring code to improve readability and future maintainability.
```
